docker/base/Dockerfile

```diff
@@ -1,5 +1,5 @@
 #FROM tomcat:7.0-jre8
-FROM tomcat:9.0.11-jre10
+FROM tomcat:9.0-jdk11-openjdk
```

Comment:
I have a general question.
"OpenJDK will have a feature release every 6 months which is only supported until the next feature release". Does it mean this tag is updated every 6 months?